app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tab/Attributes.php

```diff
@@ -161,8 +161,7 @@ protected function _getAdditionalElementTypes()
     {
         return [
             'image' => Mage::getConfig()->getBlockClassName('adminhtml/catalog_category_helper_image'),
-            'textarea' => Mage::getConfig()->getBlockClassName('adminhtml/catalog_helper_form_wysiwyg'),
-            'boolean' => Mage::getConfig()->getBlockClassName('eav/adminhtml_helper_form_boolean'),
```

Comment:
boolean doesn't exist anymore?

Reply:
I moved it to `lib/Varien/Data/Form/Element/Boolean.php`, along with adding a `Number.php` type there too. 

Since they're such basic types, It seemed to make more sense than to add those as additional element types manually. It also seemed to make more sense than creating a new folder `Mage/Eav/Block/Adminhtml/Helper` to put these basic types when they're better suited in the Varien lib.

The line I deleted was added in previously in this PR, so it's not breaking anything.